src/Microsoft.Kusto.ServiceLayer/Scripting/SmoScriptingOperation.cs

```diff
@@ -73,17 +72,10 @@ protected virtual void SetCommonEventProperties(ScriptingEventParams parameters)
             parameters.OperationId = this.OperationId;
         }
 
-        protected string GetServerNameFromLiveInstance(string connectionString, string azureAccessToken)
+        protected string GetServerNameFromLiveInstance()
         {
-            string serverName = string.Empty;
-            
-            using(var dataSource = _dataSourceFactory.Create(DataSourceType.Kusto, connectionString, azureAccessToken))
```

Comment:
Use existing datasource instead of creating a new one _just_ to get the server name